JavaScript/JavaScriptSDK/Initialization.ts

```diff
@@ -130,6 +130,7 @@ module Microsoft.ApplicationInsights {
             config.verboseLogging = Util.stringToBoolOrDefault(config.verboseLogging);
             config.emitLineDelimitedJson = Util.stringToBoolOrDefault(config.emitLineDelimitedJson);
             config.diagnosticLogInterval = config.diagnosticLogInterval || 10000;
+            config.samplingPercentage = config.samplingPercentage || 100;
```

Comment:
In .Net Sdk there is no check on sampling percentage set by customer. If it is set to a value <=0 that is equivalent to disabling telemetry collection. Value of > 100 gives the same behavior as 100. We probably want to have that in sync...